From f3b5e21d9d67136f0a6e06b532c3cab5e70095e6 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Fri, 16 Oct 2020 07:11:03 -0500
Subject: [PATCH 66/71] arm64: dts: beacon-imx8mm/imx8mn kits:  Enable FEC
 reset GPIO

The code knows when it needs to put the ethernet controller in and
out of reset, but the device tree hasn't been configured to enable
the reset GPIO on GPIO4_IO22.

This patch configures the reset pin.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi | 1 +
 arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
index 0948c4c020f4..2b4fc33276c8 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-som.dtsi
@@ -24,6 +24,7 @@
 	phy-mode = "rgmii-id";
 	phy-handle = <&ethphy0>;
 	phy-supply = <&buck6_reg>;
+	phy-reset-gpios = <&gpio4 22 GPIO_ACTIVE_LOW>;
 	fsl,magic-packet;
 	status = "okay";
 
diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi
index fb20facb5c62..e3b057499d28 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mn-som.dtsi
@@ -24,6 +24,7 @@
 	phy-mode = "rgmii-id";
 	phy-handle = <&ethphy0>;
 	phy-supply = <&buck6_reg>;
+	phy-reset-gpios = <&gpio4 22 GPIO_ACTIVE_LOW>;
 	fsl,magic-packet;
 	status = "okay";
 
-- 
2.17.1

